pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -1967,6 +2016,20 @@ typedef struct pjsua_callback
      */
     void (*on_media_event)(pjmedia_event *event);
 
+    /**
+     * This callback is called when an incoming call is rejected.
```

Comment:
Fix the doc.

Reply:
The callback is now called only when it's internally rejected by the library, right? Will using `pjsua_call_answer()` also trigger the callback?

Reply:
Minor:
make it clear in the doc that the callback is only called for calls that are internally rejected by the library. And the callback won't be called if user rejects the call using `pjsua_call_answer/answer2()`.

Reply:
No, the callback will not trigger when `pjsua_call_answer()` is used to reject the call